src/rp/index.ts

```diff
@@ -165,7 +168,18 @@ export class RelyingPartySolution {
       })
       .toSign();
 
-    return vp_token;
+    const [, vc_scope] = requestObj.payload.scope;
+    const presentation_submission = {
+      definition_id: "32f54163-7166-48f1-93d8-ff217bdb0653",
+      id: "04a98be3-7fb0-4cf5-af9a-31579c8b0e7d",
```

Comment:
This value should be randomized
```suggestion
      id: `${uuid.v4()}`,
```

Reply:
Done in https://github.com/pagopa/io-react-native-wallet/pull/22/commits/4a496e41c67328e29f1724bee30e7451c833b242

Reply:
uuid.v4() is of type number[]. I think you have to use a string. Also you imported `uuid` twice

Reply:
Done in https://github.com/pagopa/io-react-native-wallet/pull/22/commits/b31fb552ee275a7149983fc73c1ecbd8d316eaa1